substrate/frame/utility/src/lib.rs

```diff
@@ -454,6 +454,36 @@ pub mod pallet {
 			let res = call.dispatch_bypass_filter(frame_system::RawOrigin::Root.into());
 			res.map(|_| ()).map_err(|e| e.error)
 		}
+
+		/// Dispatches a function call with a provided origin.
+		/// Almost the same as [`Pallet::dispatch_as`] but check the result of the call.
```

Comment:
Maybe we can be more explicit and say it will return an error if the call returns an error.
I find the verb "check" a bit vague here.